enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1321,6 +1322,7 @@ func (c *FirecrackerContainer) dialVMExecServer(ctx context.Context) (*grpc.Clie
 			if err := c.parseFatalInitError(); err != nil {
 				return nil, err
 			}
+			return nil, status.DeadlineExceededErrorf("Failed to connect to firecracker VM exec server: %s", err)
```

Comment:
```suggestion
			// Intentionally not returning DeadlineExceededError here since it is not
			// a Bazel-retryable error, but this particular timeout should be retryable.
```

Reply:
So `InternalError` is retryable? I changed this because locally [this test](https://github.com/buildbuddy-io/buildbuddy/blob/09449bd7ae68739d91b46188da851735d83e79ae/enterprise/server/remote_execution/containers/firecracker/firecracker_test.go#L855-L857) wasn't passing, and was returning an `InternalError` that said a `ctx` deadline was exceeded, any idea how best to fix that?

Reply:
aha, lengthened a timeout in the test and it fixed everything.